src/main/java/lotto/model/Reward.java

```diff
@@ -31,13 +28,13 @@ private int getMatchCount() {
         return this.matchCount;
     }
 
-    public static Reward getReward(WinningState winningState) {
-        if (winningState.isSpecificPrizeState(SECOND_PRIZE_COUNT_CONDITION, SECOND_PRIZE_BONUS_BALL_CONDITION)) {
+    public static Reward getReward(int matchCount, boolean bonusBall) {
+        if (matchCount == SECOND_PRIZE.matchCount && bonusBall == SECOND_PRIZE.bonusBall) {
```

Comment:
bonusBall은 boolean type이고, boolean type의 변수명에 대해 고민해보면 좋을 것 같아요😃 
([좋은 코드를 위한 자바 변수명 네이밍 - 불린 변수의 네이밍 부분 ](https://woowacourse.github.io/javable/post/2020-04-24-variable_naming/) 글을 참고해보면 좋을 것 같아요. 지금의 변수명이 어떤 문제가 있는지 생각해보고 간단하게라도 코멘트 달아주세요 🙏


```suggestion
        if (matchCount == SECOND_PRIZE.matchCount && bonusBall) {
```
위와 같이 변경해도 괜찮지 않을까요? 🤔 그렇다면 `bonusBall` 필드는 필요가 있을까요?

Reply:
`bonusBall`-> `isBonusBallMatched` 로 변경을 했습니다.

`bonusBall` 필드는 2등과 3등을 구분하기 위해 필요한데... 지금 당장 Reward 반환 로직에서는 필요가 없어 보이네요 
그냥 과감히 제거해도 될까요?(현재 두개의 구분은 메소드 내부의 조건 로직에서만 구분되고 있습니다.)

Reply:
음... 저는 지금의 로직에서 bonusBall 일치 여부를 필드로 가질 필요는 없다고 생각됩니다. 
물론 상황에 따라 바뀔 수 있다는 점도 알아주세요! 😃

Reply:
넵! 👍